webrtc.html

```diff
@@ -11342,148 +11342,170 @@ <h3>Call Flow Browser to MCU</h3>
-            <th>Term/Notation</th>
-            <th>Section in [[!ECMASCRIPT-6.0]]</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr>
-            <td>Type(X)</td>
-            <td>6</td>
-          </tr>
-          <tr>
-            <td>intrinsic object</td>
-            <td>6.1.7.4</td>
-          </tr>
-          <tr>
-            <td>[[\ErrorData]]</td>
-            <td>19.5.1</td>
-          </tr>
-          <tr>
-            <td>internal slot</td>
-            <td>6.1.7.2</td>
-          </tr>
-          <tr>
-            <td>NewTarget</td>
-            <td>various uses, but no definition</td>
-          </tr>
-          <tr>
-            <td>active function object</td>
-            <td>8.3</td>
-          </tr>
-          <tr>
-            <td>OrdinaryCreateFromConstructor()</td>
-            <td>9.1.14</td>
-          </tr>
-          <tr>
-            <td>ReturnIfAbrupt()</td>
-            <td>6.2.2.4</td>
-          </tr>
-          <tr>
-            <td>Assert</td>
-            <td>5.2</td>
-          </tr>
-          <tr>
-            <td>String</td>
-            <td>4.3.17-19, depending on context</td>
-          </tr>
-          <tr>
-            <td>PropertyDescriptor</td>
-            <td>6.2.4</td>
-          </tr>
-          <tr>
-            <td>[[\Value]]</td>
-            <td>6.1.7.1</td>
-          </tr>
-          <tr>
-            <td>[[\Writable]]</td>
-            <td>6.1.7.1</td>
-          </tr>
-          <tr>
-            <td>[[\Enumerable]]</td>
-            <td>6.1.7.1</td>
-          </tr>
-          <tr>
-            <td>[[\Configurable]]</td>
-            <td>6.1.7.1</td>
-          </tr>
-          <tr>
-            <td>DefinePropertyOrThrow()</td>
-            <td>7.3.7</td>
-          </tr>
-          <tr>
-            <td>abrupt completion</td>
-            <td>6.2.2</td>
-          </tr>
-          <tr>
-            <td>ToString()</td>
-            <td>7.1.12</td>
-          </tr>
-          <tr>
-            <td>[[\Prototype]]</td>
-            <td>9.1</td>
-          </tr>
-          <tr>
-            <td>%Error%</td>
-            <td>19.5.1</td>
-          </tr>
-          <tr>
-            <td>Error</td>
-            <td>19.5</td>
-          </tr>
-          <tr>
-            <td>%ErrorPrototype%</td>
-            <td>19.5.3</td>
-          </tr>
-          <tr>
-            <td>Object.prototype.toString</td>
-            <td>19.1.3.6</td>
-          </tr>
-        </tbody>
-      </table>
-    </section>
+    <p>Some operations throw or fire <code>RTCError</code>. This is an extension
+    of <code>DOMException</code> that carries additional WebRTC-specific
+    information.</p>
     <section>
-      <h2><dfn>RTCError</dfn> Object</h2>
+      <h3><dfn>RTCError</dfn> Interface</h3>
+      <div>
+        <pre class="idl">
+          [
+              Exposed=Window,
+              Constructor(DOMString message, RTCErrorInit init)
+          ] interface RTCError /*: DOMException*/ {
```

Comment:
I get a respec error if I do ": DOMException" and I don't know why. DOMException is an interface and you should be able to extend it. Commented it out to make respec happy.

Reply:
You get an error because DOMException is not defined in the document.

Reply:
You probably have to add "DOMException is defined in <ref>" to the terminology section up front.

Reply:
In other places in the spec, including in WebIDL (example: `callback RTCPeerConnectionErrorCallback = void (DOMException error);`), it doesn't complain about it and the link produce takes you to the interface definition of another spec.

Our list of terminology has 
"When referring to exceptions, the terms throw and create are defined in [WEBIDL-1]."
which is referring to a document whose DOMException is old: https://www.w3.org/TR/2016/REC-WebIDL-1-20161215/#dfn-DOMException.

This is the reference we want: https://heycam.github.io/webidl/#idl-DOMException

Reply:
Should we update the Terminology section in a separate PR or in this one? Our WebIDL reference might affect other references in the document, and I assume we don't want to have multiple references to different versions of WebIDL.

Reply:
Definitely need to consult @dontcallmedom, and we should do it in a separate PR.